app/views/organisations/toggle_active.html.erb

```diff
@@ -0,0 +1,29 @@
+<% title = "#{action.humanize} #{@organisation.name}" %>
+<% content_for :title, title %>
+
+<% content_for :before_content do %>
+  <%= govuk_back_link(
+        href: organisation_path(@organisation),
+      ) %>
+<% end %>
+
+<%= form_for(@organisation, as: :organisation, html: { method: :patch }) do |f| %>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-l">
+        <span class="govuk-caption-l"><%= @organisation.name %></span>
+        Are you sure you want to <%= action %> this organisation?
```

Comment:
I've tried to keep the use I18n consistent with the other similar pages, but it's worth flagging up that some text that currently isn't parameterised probably could be.